sonic_platform_base/module_base.py

```diff
@@ -262,28 +266,30 @@ def get_state_info(self):
         """
         raise NotImplementedError
 
-    def get_bus_info(self, module_name):
+    def get_bus_info(self):
```

Comment:
@vvolam  suggest to use get_pci_bus_info() since there can be many bus

Reply:
@prgeor Done. Thanks!